docker/config/test.env

```diff
@@ -21,7 +21,8 @@ resource.boto.bucket_name=crashstats-test
 TELEMETRY_BUCKET_NAME=telemetry-test
 
 # Elasticsearch configuration
-ELASTICSEARCH_URLS=http://example.com:9123
```

Comment:
This allows us to test against the actual Elasticsearch service in the local dev environment rather than mocking everything all the time. Switching to testing against a real Elasticsearch will save our :bacon: when we go to upgrade.